### PR TITLE
ibc-15: Port single hop ChanOpenConfirm test cases to multihop test

### PR DESCRIPTION
**Pull request by georgeverbitskyp**
This code change ports single hop test cases for ChanOpenConfirm to multihop test

closes: ibc-15

Apparently something was not quite right with empty lines in my previous code change. This time I used gofmt on the file with code changes.

### CODE REVIEW

File: modules/core/04-channel/keeper/multihop_handshake_test.go [hunk 1]
```diff
@@ -490,12 +491,106 @@ func (suite *MultihopTestSuite) TestChanOpenConfirmMultihop() {
...
+			proofHeight = suite.Z().Chain.LastHeader.GetHeight()
+			err = suite.A().ChanOpenAck(proofHeight)
+			suite.Require().NoError(err)
+
+			channelCap = suite.Z().Chain.GetChannelCapability(suite.Z().ChannelConfig.PortID, suite.Z().ChannelID)
+
+			heightDiff = 3
+		}, false},
+		{"channel verification failed", func() {
+			// chainA is INIT, chainB in TRYOPEN
```
Comment (dshiell) [inline modules/core/04-channel/keeper/multihop_handshake_test.go:557]: maybe update this comment

File: modules/core/04-channel/keeper/multihop_handshake_test.go [hunk 2]
```diff
@@ -507,8 +602,12 @@ func (suite *MultihopTestSuite) TestChanOpenConfirmMultihop() {
...
 
 			err = suite.Z().Chain.App.GetIBCKeeper().ChannelKeeper.ChanOpenConfirm(
-				suite.Z().Chain.GetContext(), suite.Z().ChannelConfig.PortID, suite.Z().ChannelID,
-				channelCap, proof, suite.Z().GetClientState().GetLatestHeight(),
+				suite.Z().Chain.GetContext(),
+				suite.Z().ChannelConfig.PortID,
+				suite.Z().ChannelID,
+				channelCap,
+				proof,
+				malleateHeight(suite.Z().GetClientState().GetLatestHeight(), heightDiff),
```
Comment (dshiell) [inline modules/core/04-channel/keeper/multihop_handshake_test.go:610]: suite.Z().ProofHeight()

**Review by dshiell — APPROVED** [review]
lgtm, thanks!

File: modules/core/04-channel/keeper/multihop_handshake_test.go (hunk 2 above)
Comment (georgeverbitskyp) [reply modules/core/04-channel/keeper/multihop_handshake_test.go]: Fixed

**Pull request merged**